javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,195 @@
+
+// Item array removal
+
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+// Write some code here
+names.splice(1, 1);
+// Code done
+
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+
+
+
+
+// When will we be there??
+
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function counterTravelTime(object) {
+    const reminder = object.destinationDistance % object.speed;
+    const minutes = Math.round(reminder * 60 / object.speed);
+    const hours = (object.destinationDistance - reminder) / object.speed;
+    return `${hours} hours and ${minutes} minutes `;
```

Comment:
Several questions to thing about:
1. let's imagine that speed === 0, would it work correctly?
2. let's imagine that we are super close to the object (travel takes less than an hour). How would it be printed, is it possible to improve output format?
3. let's image that trip takes 1 hour and 32 minutes. but we'll print `hours` in plural form